src/pet.js

```diff
@@ -0,0 +1,88 @@
+const INCREASE_AGE = 1;
+const MAXIMUM_AGE = 30;
+
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_FITNESS = 0;
+const INCREASE_FITNESS = 4;
+const DECREASE_FITNESS = 3;
+const CHECKUP_FITNESS = 3;
+
+const MINIMUM_HUNGER = 0;
+const MAXIMUM_HUNGER = 10;
+const DECREASE_HUNGER = 3;
+const INCREASE_HUNGER = 5;
+const CHECKUP_HUNGER = 5;
+
+const ERROR_MESSAGE = 'Your pet is no longer alive :('
```

Comment:
Great to see so many constants here! Would be even more useful if they were to be stored in their own file and imported in, as this would allow them to be used in the test files as well. Thats a bit of an extra mile kind of thing, but thought it was worth suggesting all the same :)